index.bs

```diff
@@ -6826,8 +6824,8 @@ This [=client extension|client=] [=registration extension=] facilitates reportin
         ::  This OPTIONAL property is a [=human palatability|human-palatable=] description of the credential's [=managing authenticator=],
             chosen by the user.
 
-            The [=client=] MUST allow the user to choose this value,
-            MAY or MAY not present that choice during [=registration ceremonies=],
+            During [=registration ceremonies=] the [=client=] MUST allow the user to choose this value,
+            MAY or MAY not present that choice,
             and MAY reuse the same value for multiple credentials with the same [=managing authenticator=] across multiple [=[RPS]=].
 
```

Comment:
Below this, the final paragraph about `[$credential record/authenticatorDisplayName$]` only applies to registration ceremonies. We should add something to cover authentication ceremonies too. How about this?

```diff
             If the [=[RP]=] includes an <code>[$credential record/authenticatorDisplayName$]</code> [=struct/item=] in [=credential records=],
             the [=[RP]=] MAY offer this value, if present,
             as a default value for the <code>[$credential record/authenticatorDisplayName$]</code> of the new [=credential record=].
+            If the {{authenticatorDisplayName}} extension output from an [=authentication ceremony=]
+            is different from the <code>[$credential record/authenticatorDisplayName$]</code> of the [=credential record=],
+            the [=[RP]=] MAY offer the user to update the <code>[$credential record/authenticatorDisplayName$]</code> of the [=credential record=].
```

Unless `authenticatorDisplayName` should be forbidden for assertions? But I can imagine it being useful for assertions too.

Reply:
That's a good suggestion, I've added something specific for post-auth next steps for an RP to react to a different value during auth.

> Unless `authenticatorDisplayName` should be forbidden for assertions? But I can imagine it being useful for assertions too.

That was the intention of this PR, extending `authenticatorDisplayName`'s use for `.get()` calls too. Maybe I'm missing something about your comment here?

Reply:
> Maybe I'm missing something about your comment here?

Sorry, you're not. I too don't think it would make sense to forbid `authenticatorDisplayName` for assertions. :laughing: